modules/secure-logging/slog.c

```diff
@@ -1084,66 +1174,53 @@ int iterateBuffer(guint64 entriesInBuffer, GString **input, guint64 *nextLogEntr
               *nextLogEntry = logEntryOnDisk;
             }
```

Comment:
this part betwen line 1130-1176 might be worth moving to a separate function like `step_log_entry` or similar